pkg/gatherers/clusterconfig/openshift_storage.go

```diff
@@ -0,0 +1,53 @@
+package clusterconfig
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
+)
+
+// GatherOpenshiftStorage collects `storageclusters.ocs.openshift.io` resources
+// from Openshift Data Foundation Stack.
+//
+// API Reference:
+//   https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
+//
+// * Location in archive: config/storage/<namespace>/<name>.json
+// * Id in config: clusterconfig/openshift_storage
+// * Since versions:
+//   * 4.11+
+func (g *Gatherer) GatherOpenshiftStorage(ctx context.Context) ([]record.Record, []error) {
+	gatherDynamicClient, err := dynamic.NewForConfig(g.gatherKubeConfig)
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	return gatherOpenshiftStorage(ctx, gatherDynamicClient)
+}
+
+func gatherOpenshiftStorage(ctx context.Context, dynamicClient dynamic.Interface) ([]record.Record, []error) {
+	loggingResourceList, err := dynamicClient.Resource(openshiftStorageResource).List(ctx, metav1.ListOptions{})
```

Comment:
It's not a logging resource right :)

Reply:
resolved by https://github.com/openshift/insights-operator/pull/614/commits/8b374350c496c48923ffb2dcce9e2eadd54f6ba0